helsinki/login/messages/messages_en.properties

```diff
@@ -123,8 +123,10 @@ locale_fi.languageLinkTitle = Suomeksi
 locale_en.languageLinkTitle = In English
 locale_sv.languageLinkTitle = På svenska
 requiredFieldIndication = Fields marked with an * are compulsory.
+windowCanBeClosed = You can close this window
 serviceDataUsageListTitle = We allow the <strong>{0}</strong> service access to the following information on your profile:
 dataHandlingNote = You can later manage access to this information or delete it from your Helsinki profile.
 readHelsinkiDocs = Please refer to the City of Helsinki’s <a href="{0}" {2}>file description</a>{3} and <a href="{1}" {2}>privacy policy</a>{3}.
 serviceDataUsageListTitleOldUser = The <strong>{0}</strong> service requests permission to access the following information on your profile:
-weUseInformationTitle = We use your information
\ No newline at end of file
+weUseInformationTitle = We use your information
+loginDataUsageNote = When you identify yourself, we use your personal data stored in the Helsinki profile and allow the x-service to access your information.
```

Comment:
This text looks like a template ("x-service") and is not used, so should be manually deleted from all translation files.

Reply:
Removed